includes/class-wc-order-data-store-custom-table.php

```diff
@@ -126,7 +126,7 @@ protected function update_post_meta( &$order ) {
 		$changes    = array();
 		$order_data = array(
 			'order_id'             => $order->get_id( 'edit' ),
-			'order_key'            => $order->get_order_key( 'edit' ),
+			'order_key'            => $order->get_order_key( 'edit' ) ?: null,
```

Comment:
[The "Elvis" operator (`expr1 ?: expr3`) is unsupported in PHP < 5.3](http://php.net/manual/en/language.operators.comparison.php#language.operators.comparison.ternary), so we need to be a bit more explicit.

[Here's the Travis CI failure, for reference](https://travis-ci.org/liquidweb/woocommerce-custom-orders-table/jobs/452712383#L667).

Reply:
@stevegrunwell Argh, I actually did go and look at the version for the Elvis operator, and when I saw PHP 5.3, figured that was sufficient; my apologies for not reading the contribution guidelines thoroughly. Will adjust.

Reply:
I do it all the time. When I'm not writing WordPress, I'm often working in Laravel apps. The number of times I've almost committed return types to this project... 😉